quarkus-integration/quarkus-jackson/deployment/src/main/java/ai/timefold/solver/quarkus/jackson/deployment/TimefoldJacksonProcessor.java

```diff
@@ -15,7 +17,14 @@ FeatureBuildItem feature() {
     }
 
     @BuildStep
-    void registerTimefoldJacksonModule(BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
+    ClassPathJacksonModuleBuildItem registerTimefoldJacksonModule() {
+        // Make timefold-solver-jackson discoverable by quarkus-rest
+        // https://quarkus.io/guides/rest-migration#service-loading
+        return new ClassPathJacksonModuleBuildItem(TimefoldJacksonModule.class.getName());
+    }
+
+    @BuildStep
+    void registerTimefoldObjectMapperCustomizer(BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
```

Comment:
In fact, this is no longer needed, as well as `TimefoldObjectMapperCustomizer`.

Reply:
I'm good with removing this.